portfolio/src/main/webapp/script.js

```diff
@@ -72,6 +97,28 @@ function injectHeader(header){
 
 }
 
+/*
+	*Gets data from {header} object and creates elements to render to html
```

Comment:
im guessing this description is copied from above?

Reply:
Yeah. Above the inject functions I give a brief description of the html element they are creating and write out how the resulting html would look like

Reply:
Yeah but are the comments the same? Do you still get data from {header} object?

Reply:
Ooh gotcha. I gets it from the repo object, not the header object. Sorry for that
Resolved